cmd/newrelic-infra/newrelic-infra.go

```diff
@@ -437,7 +434,7 @@ func initializeAgentAndRun(c *config.Config, logFwCfg config.LogForward) error {
 		FluentBitExePath:     c.FluentBitExePath,
 		FluentBitNRLibPath:   c.FluentBitNRLibPath,
 		FluentBitParsersPath: c.FluentBitParsersPath,
-		FluentBitVerbose:     c.Verbose != 0 && trace.IsEnabled(trace.LOG_FWD),
```

Comment:
@brushknight Any idea why `FluentBitVerbose` was enabled only if the LOG_FWD feature trace was enabled?

Reply:
I suspect as this is what that trace tracing